cmd/clusterctl/client/cluster/template_test.go

```diff
@@ -310,7 +310,7 @@ func Test_templateClient_GetFromURL(t *testing.T) {
 	// redirect stdin
 	saveStdin := os.Stdin
 	defer func() { os.Stdin = saveStdin }()
-	os.Stdin, err = os.Open(path)
+	os.Stdin, err = os.Open(path) //nolint:gosec
```

Comment:
I was surprised to see the gosec linter fail on this line since it wasn't changed in the pr. I nolint'ed it since it's a test anyway. But it's odd